app/javascript/packs/public.js

```diff
@@ -75,6 +87,36 @@ function main() {
       content.textContent = formattedDate;
     });
 
+    const isToday = date => {
+      const today = new Date();
+
+      return date.getDate() === today.getDate() &&
+        date.getMonth() === today.getMonth() &&
+        date.getFullYear() === today.getFullYear();
+    };
+
+    [].forEach.call(document.querySelectorAll('time.relative-formatted'), (content) => {
+      const datetime = new Date(content.getAttribute('datetime'));
+
+      let title;
+      let textContent;
+
+      if (isToday(datetime)) {
+        const formattedTime = timeFormat.format(datetime);
+
+        title = content.title.replace('%{time}', formattedTime);
+        textContent = content.textContent.replace('%{time}', formattedTime);
```

Comment:
I think it would be best to format the string from scratch instead of replacing the placeholder part.

Reply:
Was expecting that. If using a new class is okay, going to use `IntlMessageFormat` instead.